### ISSUE TITLE
Clarification on how to use universal with an Express application

### ISSUE THREAD

**Issue by qdevial** [angular/universal#1638]
Hello, I was wondering if I could get an input about the optimal architecture with Universal and Express.

Previously, I had my main Express application which did not compile with the angular base code and I was just using the express engine to deliver the server build of my angular application on the fly when needed, and that was great.

Seems like now it is no longer possible and you need to use a universal server like a server on its own which handles certain kinds of requests to deliver the application.

So basically you need a server for your API and a Universal server to deliver the angular application and a way to proxy a request depending on what you need ( data or the application ), is that correct ?

But how would you manage for example a connected user,do you need to duplicate the code to extract the session id or the jwt token from the cookie of the request on both servers ?

**Comment by qdevial** [comment]
I found out how to proceed to use an existing express application on this issue : https://github.com/angular/universal/issues/1363 I think it would be a great addition to add this possibility in the documentation somewhere.

**Comment by CaerusKaru** [comment]
This is not something that the Universal team is going to take on. My recommendation is to take your learnings and write a Medium post. We'd be more than happy to promote it and add links from our docs!
